CONTRIBUTING.md

```diff
@@ -23,6 +23,16 @@ Create a new card in the **No Status** column describing your contribution and t
 
     * The [InnerSource Commons Slack] [#ispo-working-group] community will post the next meeting.
 
+## New Content
+
+When contributing new content, it can go into one of a few locations:
+
+* If it can be represented as a pattern, then contribute it to the [InnerSource Patterns] project.
+Be sure to check if there is an existing pattern that you can just update.
+* If it can be represented as a training segment, then contribute it to the [InnerSource Learning Path] project.
+* Documents that don't fit either of these categories (e.g. graphics, opinionated document, etc.) can go directly this `ispo-working-group` repo as markdown.
+If your content cannot be represented in markdown (or you are not comfortable working in the markdown) then you can put it in the [ISPO Working Group GDrive].
```

Comment:
```suggestion
If your content cannot be represented in markdown (or you are not comfortable working in markdown) then you can put it in the [ISPO Working Group GDrive].
```

Reply:
Should we add here that we may take this content and convert it to markdown ourselves, to allow for versioned updates in the future? Or maybe it is too early to say if we really do that?